release/rllib_tests/learning_tests/hard_learning_tests.yaml

```diff
@@ -461,13 +461,18 @@ slateq-interest-evolution-recsim-env:
     env: ray.rllib.examples.env.recommender_system_envs_with_recsim.InterestEvolutionRecSimEnv
     run: SlateQ
     stop:
-        episode_reward_mean: 162.0
+        episode_reward_mean: 155.0
```

Comment:
Oh no, this is not good :(
155.0 is random performance. Did this test not do any better? It should reach the 162.0 reward easily.